cookbooks/bcpc-hadoop/recipes/core_site.rb

```diff
@@ -56,6 +56,8 @@
 
      'hadoop.security.group.mapping.ldap.search.attr.group.name' =>
        'cn',
+     'hadoop.security.group.mapping.ldap.search.group.hierarchy.levels' =>
+       '1',
```

Comment:
Do we want this value to be configurable through an attribute? or `1` is sufficient assuming that that is the max number of hierarchy levels in a typical installation?